### PR TITLE
Create ENAH-author-date

### PR DESCRIPTION
**Pull request by natch-nacht**
This is the author-date CSL  file for Escuela Nacional de Antropología e Historia in Mexico City, based on the new Editorial Standards of the school. They were developed to facilitate the clarity of critical apparatus in  publications (books, manuals, magazines and bulletins), theses and homework, using bibliographic managers such as Mendeley.

### CODE REVIEW

**Comment by adam3smith** [comment]
Thanks. Please don't make any more pull requests. We'll review this and let you know what and how to fix, but that can all be done within a single pull request, so don't create a new one every time you're trying to correct an error.

**Comment by natch-nacht** [comment]
Sorry, my Wi-Fi failed and I thought the pull request was failed to.

--
Enviado desde mi iMac

**Comment by adam3smith** [comment]
are there different versions of this style? We'd only add autor-fecha/author-date if there's also a numeric and/or a note version.

You have two macros you're not using ("container-contributors", "recipient") -- delete them.

(there's also a typo in the self link, but since that may change, don't worry about that).

**Comment by adam3smith** [comment]
You can update your pull request by using the pencil button here: https://github.com/natch-nacht/styles/blob/patch-5/escuela-nacional-de-antropologia-e-historia-author-date.csl

**Comment by natch-nacht** [comment]
It is the same version, I pull it several times because my Wi-Fi failed and thought it was not sent.

I have other versions for the note variants, but I need to validate them first.

I will review these two macros you mention me.

Regards.

Enviado desde mi iPhone

**Comment by adam3smith** [comment]
@rmzelle -- do you want to go with author-date or autor-fecha in the title?

**Comment by natch-nacht** [comment]
author-date is fine, there will also be a CSL file for short-note and full-note variants.

--
Enviado desde mi iMac

File: escuela-nacional-de-antropologia-e-historia-author-date.csl [hunk 1]
```diff
@@ -0,0 +1,513 @@
...
+      <uri>http://www.enah.edu.mx/index.php/difu-cul/publicaciones</uri>
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="anthropology"/>
+    <category field="history"/>
+    <summary>Estilo de citas de la Escuela Nacional de Antropología e Historia -- variante autor-fecha</summary>
+    <updated>2015-06-25T20:21:07-06:00</updated>
+    <!--ISSN revista Cuicuilco--> 
+    <issn>1405-7778</issn>
+  </info>
```
Comment (adam3smith) [inline escuela-nacional-de-antropologia-e-historia-author-date.csl]: yeah, so let's remove the ISSN here as discussed.

**Comment by rmzelle** [comment]
Looks good to me.

**Pull request merged**